### PR TITLE
Adds radiation damage to nuclear bombs

### PR DESCRIPTION
**Pull request by TemporalOroboros**
## About the PR 
Makes the nuclear bomb explosion also deal radiation damage.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
No visible changes.

**Changelog**

:cl:
- add: The nuclear bomb now deals radiation damage in addition to the normal explosion damage.

### CODE REVIEW

**Comment by Just-a-Unity-Dev** [comment]
I feel as it should also use the RadiationSystem to radiate the affected areas

**Comment by metalgearsloth** [comment]
> I feel as it should also use the RadiationSystem to radiate the affected areas

Does radiation even work off grid.

File: Resources/Prototypes/explosion.yml [hunk 1]
```diff
@@ -58,6 +58,21 @@
...
+      Heat: 5
+      Blunt: 5
+      Piercing: 5
+      Radiation: 5
+  tileBreakChance: [0, 0.5, 1]
+  tileBreakIntensity: [0, 10, 30]
+  tileBreakRerollReduction: 20
+  lightColor: Orange
+  texturePath: /Textures/Effects/fire.rsi
+  fireStates: 3
```
Comment (metalgearsloth) [inline Resources/Prototypes/explosion.yml:74]: Does this really need a new damage type? This adds overhead to the explosion tiles for something that is supposed to destroy everything in range anyway.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by Macoron** [comment]
> > I feel as it should also use the RadiationSystem to radiate the affected areas
> 
> Does radiation even work off grid.

It does. You could spawn a big radiation source, with insane radiation intensity to feel nuke explosion zone. 

Though I'm not sure why would you do that if round will end in one minute and everything in radius supposed to be dead anyway.

**Review by EmoGarbage404 — COMMENTED** [review]
feels like unnecessary overhead considering that a nuke triggers a round end. Besides that, just doing radiation damage isn't really interesting enough to warrant the overhead from a new explosion type. If you're going to do anything, do radioactive fallout, but then consider that adding that is kinda pointless given that you're irradiating a pile of space debris.

**Comment by ElectroJr** [comment]
It seems like sloth emo and me are all of the opinion that this is unnecessary and will just contribute to worse nuke/explosion performance, so closing this.

**Pull request closed without merging**